Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs

```diff
@@ -44,7 +45,7 @@ void Update()
             muzzleFlashTimer += Time.deltaTime;
         }
         coolDownTimer += Time.deltaTime;
-        cooldownBar.SetFill(coolDownTimer, currGun.cooldownTime);
+        // cooldownBar.SetFill(coolDownTimer, currGun.cooldownTime);
```

Comment:
Is this in final commit? I want cooldown bar to be in playergeneralshootcontroller for main game

Reply:
commented this coolDowBar.setFill code back in, not sure how/why it was commented-out